scripts/flattener.py

```diff
@@ -341,7 +342,11 @@ def gather_objects(input_object, start_type=None):
 def get_value(obj, prop):
 	path = prop.split('.')
 	if len(path) == 1:
-		return obj.get(prop, unreported_value)
+		if path[0] == '@type':
```

Comment:
We just need to remember that if we end up ever adding `@type` to any of the other object types (donor, tissue section, suspension, library, raw_matrix, or seq_run), this code will need to have a more specific if clause to distinguish.